src/AutoBuild.jl

```diff
@@ -633,9 +647,8 @@ function autobuild(dir::AbstractString,
     # We must prepare our sources.  Download them, hash them, etc...
     source_files = download_source.(sources; verbose=verbose)
 
-    # Our build products will go into ./products
-    out_path = joinpath(dir, "products")
-    try mkpath(out_path) catch; end
+    # Our build products will go into products_dir (usually ./products)
+    try mkpath(products_dir) catch; end
```

Comment:
I know you didn't introduce this line, but I don't think we want to wrap `mkpath` with a `try`/`catch` block :thinking:

Reply:
Going back in Git blame, it looks like it used to be a very common pattern, but it looks like this is the last remaining relic. I can remove the `try ... catch`.